src/encord_active/lib/encord/actions.py

```diff
@@ -159,6 +171,14 @@ def create_project(
         for counter, new_label_row in enumerate(all_new_label_rows):
             initiated_label_row: dict = new_project.create_label_row(new_label_row["data_hash"])
             original_data = dataset_creation_result.du_original_mapping[new_label_row["data_hash"]]
+
+            new_label_row_hash = initiated_label_row["label_hash"]
+            new_data_unit_hash = dataset_creation_result.lr_du_mapping[
+                (original_data["label_row_hash"], original_data["data_unit_hash"])
+            ][1]
```

Comment:
I know this isn't part of your changes, but currently `lr_du_mapping` is typed as a dictionary mapping a tuple to a tuple. This is slightly hard to read when we access it with `[1]` or `[0]`, could you change it to `NamedTuple`s?

Something like:
```python
class LabelRowDataUnit(NamedTuple):
    label_row: str
    data_unit: str
```

so we could change the statement to 
```suggestion
            new_data_unit_hash = dataset_creation_result.lr_du_mapping[
                (original_data["label_row_hash"], original_data["data_unit_hash"])
            ].data_unit
```